lib/collections/initiatives.js

```diff
@@ -21,8 +21,8 @@ if(Meteor.isServer) {
   Initiatives._ensureIndex({title: 1, category: 1});
 }
 
-InitiativeCategories = ['Event', 'Charity', 'Non Profit', 'Open Source'];
-InititaiveCategorySlugs = InitiativeCategories.map(function(category) {
+Initiatives.InitiativeCategories = ['Event', 'Charity', 'Non Profit', 'Open Source'];
```

Comment:
This is fine and I actually like attaching it to the Initiatives Collection as like a static property. However, this will effect the eslint stuff from the other branch. It shouldn't cause errors, but in ESLint we have to specify our globals and `InitiativeCategories` and `InitiativeCategorySlugs` were globals. So they will have to be removed from the eslint config.